data_pipeline/opensearch/bigquery_to_opensearch_pipeline.py

```diff
@@ -78,10 +80,102 @@ def get_opensearch_client(opensearch_target_config: OpenSearchTargetConfig) -> O
     )
 
 
+def get_opensearch_ingest_pipeline_tests_body(
```

Comment:
I was thinking should we have a separated module for these functions as they are for testing and not effecting directly the opensearch pipeline. Maybe undersame forlder but specifically for these functions?

Reply:
The only thing with naming the the module something like `opensearch_ingest_pipeline_testing` is that the test module would then be called `opensearch_ingest_pipeline_testing_test`. What do you think?

Alternatively we could have a module for ingest pipeline related code, e.g. `opensearch_ingest_pipeline` or just `ingest_pipeline`?

Reply:
I opted for `ingest_pipeline` for now but could revisit and further enhance.